mysql-test/suite/rpl/t/rpl_bug-lp-1667307.test

```diff
@@ -0,0 +1,51 @@
+#
+# In the case if the bug is not fixed the test will crash on assert
+# in Rows_query_log_event::do_apply_event().
+#
+--source include/have_debug.inc
```

Comment:
Yes, we need it, because the "assert", on which the test crashes, if the bug is not fixed, happens only on debug build as "DBUG_ASSERT" macros is used. See the following code:

```c++
int Rows_query_log_event::do_apply_event(Relay_log_info const *rli)
{
  ...
  DBUG_ASSERT(rli->rows_query_ev == NULL);
...
}
```